public/app/percona/dbaas/components/DBCluster/DBClusterStatus/DBClusterStatus.tsx

```diff
@@ -64,16 +76,24 @@ export const DBClusterStatus: FC<DBClusterStatusProps> = ({ dbCluster, setSelect
       {showProgressBar ? (
         <ProgressBar
           status={statusError ? ProgressBarStatus.error : ProgressBarStatus.progress}
-          finishedSteps={finishedSteps || 0}
-          totalSteps={totalSteps || 0}
+          finishedSteps={finishedSteps}
+          totalSteps={totalSteps}
           message={getProgressMessage(status, prevStatus.current)}
           dataTestId="cluster-progress-bar"
         />
-      ) : (
-        <span className={cx(styles.status, statusStyles)} data-testid={`cluster-status-${STATUS_DATA_QA[status]}`}>
-          {Messages.dbcluster.table.status[status]}
+      ) : status ? (
+        <span
+          className={cx(styles.status, statusStyles)}
+          data-testid={`cluster-status${status ? `-${STATUS_DATA_QA[status]}` : ''}`}
+        >
+          {status ? Messages.dbcluster.table.status[status] : ''}
```

Comment:
Status will always be truthy here (see conditional rendering above).

Reply:
fixed